extensions/2.0/Vendor/EXT_feature_metadata/examples/microcosm/microcosm.gltf

```diff
@@ -0,0 +1,365 @@
+{
+    "asset" : {
+        "generator" : "Khronos glTF Blender I/O v0.9.36",
+        "version" : "2.0"
+    },
+    "extensionsUsed": [
+        "EXT_feature_metadata"
+    ],
+    "extensions": {
+        "EXT_feature_metadata": {
+            "extras": {
+                "draftVersion": "0.0.0"
+            },
+            "featureClasses": {
+                "NDVI": {
+                    "name": "Normalized difference vegetation index",
+                    "properties": {
+                        "NDVI": {
+                            "valueType": "UINT8",
+                            "normalized": true
+                        }
+                    }
+                },
+                "LULC": {
+                    "name": "Landcover/Landuse",
+                    "properties": {
+                        "Name": {
+                            "name": "Name",
+                            "valueType": "STRING"
+                        },
+                        "Color": {
+                            "name": "Color",
+                            "elementType": "ARRAY",
+                            "valueType": "UINT8",
+                            "valueCount": 3
+                        }
+                    }
+                }
+            },
+            "featureCollections": {
+                "LandCover": {
+                    "featureClass": "LULC",
+                    "featureCount": 6,
+                    "properties": {
+                        "Name": {
+                            "elementCount": 256,
+                            "bufferView": 5,
+                            "offsetsBufferView": 6
+                        },
+                        "Color": {
+                            "elementCount": 256,
+                            "bufferView": 7
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "scene" : 0,
+    "scenes" : [
+        {
+            "name" : "Scene",
+            "nodes" : [
+                0,
+                1,
+                2,
+                3
```

Comment:
Oh, I should probably collapse these back into a single mesh...

Reply:
Though... until we have some sort of tool to make use of this metadata, it's a lot easier to visualize the textures this way...

Reply:
after talking with @lilleyse, I ended up removing these debug meshes.